src/app/my-regular-giving/my-regular-giving.component.html

```diff
@@ -15,16 +15,27 @@
         @if (mandates.length > 0) {
           <biggive-grid columnCount="2">
             @for (mandate of mandates; track mandate.id) {
-              <biggive-basic-card
-                [icon]="false"
-                [textColour]="'primary'"
-                [mainImageUrl]="''"
-                [mainTitle]="mandate.charityName"
-                subtitle="{{(mandate.amount.amountInPence / 100) | exactCurrency:mandate.amount.currency }}, {{ordinal(mandate.schedule.dayOfMonth)}} of each month, {{mandate.giftAid ? 'with Gift Aid, ' : ''}} since {{ mandate.schedule.activeFrom | date: 'mediumDate'}}"
-                [buttonUrl]="'/campaign/' + mandate.campaignId"
-                [buttonAlign]="'center'"
+              <biggive-container-card
+                card-colour="white"
+                background-colour="primary"
               >
-              </biggive-basic-card>
+                <div class="mandate">
+                  <h3><a href="{{'/campaign/' + mandate.campaignId}}">{{ mandate.charityName }}</a></h3>
+                <ul>
+                  <li>Amount: {{(mandate.amount.amountInPence / 100) | exactCurrency:mandate.amount.currency }}</li>
+                  @if (mandate.giftAid) {
```

Comment:
@NoelLH We talked about moving the GA decision info off the mandate and onto the donor's person record, but I guess if we do that its still useful to do the multiplication by 1.25 for display. More reason to move that calculation into matchbot.